docs/en/transform-v2/embedding.md

```diff
@@ -0,0 +1,197 @@
+# Embedding
```

Comment:
How about renamed to `EmbeddingRemote`? So that we can support `EmbeddingLocal` in the future.

Reply:
get